config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   get "/healthcheck", to: proc { [200, {}, ["OK"]] }
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  get "/check-link", to: proc { [200, {}, [{}.to_json]] }
```

Comment:
Yeah, that's a good point, I'm not sure on that either - calling it just `/check` does limit the scope of it given it's a "checker" which could be argued to be either a good or bad thing. Also unsure if it should be "link" or 'uri" I suppose since we have a query string of `uri=` it does make it more redundant